### PR TITLE
Indicate problematic class when transform fails

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: src/main/java/org/pantsbuild/jarjar/util/JarTransformer.java [hunk 1]
```diff
@@ -35,7 +35,11 @@ public boolean process(EntryStruct struct) throws IOException {
             }
 
             GetNameClassWriter w = new GetNameClassWriter(ClassWriter.COMPUTE_MAXS);
-            reader.accept(transform(w), ClassReader.EXPAND_FRAMES);
+            try {
+                reader.accept(transform(w), ClassReader.EXPAND_FRAMES);
+            } catch (Exception e) {
```
Comment (jsirois) [inline src/main/java/org/pantsbuild/jarjar/util/JarTransformer.java:40]: It looks like both the catch and throw can be narrower. Can the catch just be for RuntimeException and ditto the throw?

Comment (mernst) [reply src/main/java/org/pantsbuild/jarjar/util/JarTransformer.java:40]: That would be fine with me, feel free to change it.
I chose `Exception` for consistency with the existing code -- it's what is used on [line 31](https://github.com/pantsbuild/jarjar/pull/37/files/00981ff19b68ac74f61af87a8cd9365c978630a4#diff-08f9a1a5acee7458caadc7fa5f7dadd5R31) just above.

Comment (jsirois) [reply src/main/java/org/pantsbuild/jarjar/util/JarTransformer.java:40]: OK - I'll take this and then follow-up with a narrowing adjustment.

**Comment by jsirois** [comment]
Thank you!

**Pull request merged**